### PR TITLE
feat(bulk): add bulk action to click2call

### PR DESCRIPTION
**Pull request by JeremyDec**
## Title of the Pull Requests

Add bulk action to click 2 call user add

(Linked to change on ovh-api-services/telephony/line/click2call/user/*.js)

### CODE REVIEW

**Comment by JeremyDec** [comment]
No deletion, it's an update, but it's still not necessary => fixing it

File: client/app/telecom/telephony/line/calls/click2Call/add-user/telecom-telephony-line-calls-click2Call-add-user.controller.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-angular.module("managerApp").controller("TelecomTelephonyLineClick2CallAddUserCtrl", function ($q, $stateParams, $state, $translate, TelephonyGroupLineClick2CallUser, Toast) {
+angular.module("managerApp").controller("TelecomTelephonyLineClick2CallAddUserCtrl", function ($q, $stateParams, $state, $translate, $timeout, OvhApiTelephony, TelephonyMediator, TelephonyGroupLineClick2CallUser, Toast, telephonyBulk) {
```
Comment (Jisay) [inline client/app/telecom/telephony/line/calls/click2Call/add-user/telecom-telephony-line-calls-click2Call-add-user.controller.js]: \- `$timeout`

File: client/app/telecom/telephony/line/calls/click2Call/add-user/telecom-telephony-line-calls-click2Call-add-user.controller.js [hunk 2]
```diff
@@ -57,4 +56,68 @@ angular.module("managerApp").controller("TelecomTelephonyLineClick2CallAddUserCt
...
+        }
+    };
+
+    self.filterServices = function (services) {
+        var filteredServices = _.filter(services, function (service) {
+            return ["sip", "mgcp"].indexOf(service.featureType) > -1;
+        });
+
+        return _.filter(filteredServices, function (service) {
+            return _.some(service.offers, _.method("includes", "sipfax")) ||
```
Comment (Jisay) [inline client/app/telecom/telephony/line/calls/click2Call/add-user/telecom-telephony-line-calls-click2Call-add-user.controller.js]: Why do we need this ? To test for voicefax `featureType` ?

Comment (JeremyDec) [reply client/app/telecom/telephony/line/calls/click2Call/add-user/telecom-telephony-line-calls-click2Call-add-user.controller.js]: It's needed to enable Click2Call action in calls management page, so I guess same tests are mandatory for bulk action

**Pull request merged**